### PR TITLE
Add main page heat map

### PR DESCRIPTION
**Pull request by pbkabali**
### What this PR does

This PR adds the greyscale heat map of the main page

### Screenshot

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,408 @@
...
+        <div></div>
+        <div></div>
+      </div>
+  </div>
+
+  <div class="header-right">
+    <input type="search" name="article" >
+  </div>
+    
+  </header>
```
Comment (krys2fa) [inline index.html]: - [ ] [OPTIONAL] The navigation bar should use nav element with ul/li inside, however, you are allowed to used div as the main focus is a heat map. (content is not important)

**Review by krys2fa — CHANGES_REQUESTED** [review]
## Nicely done @pbkabali and @ABDELLANI-Youcef 👏 
Your project has a professional and descriptive README and displays good use of semantic HTML. 💪 
Just a few more adjustments to complete the project.
### Status: Changes Required ♻️ 

- [MANDATORY] Make sure to build your layout using a width of 1440px on the original page. You can use the Developer Tools to match that screen size. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/design_teardown/pageLayoutScreenSize.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540025/)) 
Currently, your project does not fit in the screen size, hence the horizontal scroll bar at the bottom of the page. Please fix this.

Please request for another review after implementing the changes.

Happy Coding 🖥️  😃 

###### You can contact me on slack (Chris Amanor) for any clarification.

**Comment by pbkabali** [comment]
Hi @krys2fa, thank you for your review. We have worked on the overflow issue and also used `nav` for the navbar. 
A new review request has been submitted.

**Comment by krys2fa** [comment]
# Status: Project Approved! 🥇 
Congratulations on making all the requested changes and getting this project approved. 👍 

You can now merge this branch into the master branch and go on with your next project.

💡  Do not forget to submit your project completion form after merging this branch.

Also, please update the live link in your README.md file so the latest changes can be easily seen.

All the best team,
Chris

**Comment by pbkabali** [comment]
Thank you very much @krys2fa

**Pull request merged**